src/modal/Modal.css

```diff
@@ -0,0 +1,88 @@
+* {
```

Comment:
We are using Styled-components, please pass all these styles to styled-components